awx/main/isolated/manager.py

```diff
@@ -187,6 +189,19 @@ def check(self, interval=None):
                                                       self.private_data_dir,
                                                       extravars=extravars)
             status, rc = runner_obj.status, runner_obj.rc
+
+            if self.check_callback is not None and not self.captured_command_artifact:
+                command_path = self.path_to('artifacts', self.ident, 'command')
+                # If the configuration artifact has been synced back, update the model
+                if os.path.exists(command_path):
```

Comment:
Nice work on this - this'll keep things working _until_ the new artifact lands in the next official runner.